### ISSUE TITLE
Release latest version

### ISSUE THREAD

**Issue by blakeembrey** [chenglou/RCSS#59]
I just tried to use this, but can't since it doesn't support flexbox. Looks like it's supported int the last dependency on `valid-css-props`, so could we get a patch released? What's the purpose of disallowing styles anyway?

**Comment by chenglou** [comment]
Yeah that'll probably a saner behaviour. I just published a new release to valid-css-props and to this. You can `npm install` it now.

Leaving this issue open to fix the style warning issue in the future.
